### ISSUE TITLE
Website homepage images missing alt attributes

### ISSUE THREAD

**Issue by vivrichards600** [SeleniumHQ/selenium-ide#530]
## 🐛 Bug Report

Images on the home page of the website are missing alt attributes which are used for accessibility.

## To Reproduce

Run the web site  locally and navigate to the home page. Right click an image and inspect it to view it's attributes, images are missing alt attributes.

## Expected behavior

Within Index.js where image properties are set, they are missing a property named imageAlt which is used to render the alt tag attribute for the images, for example:

            title: 'Resilient Tests',
            content:
              'Selenium IDE records multiple locators for each element it interacts with. If one locator fails during playback, the others will be tried until one is succesful.',
            image: `${baseUrl}img/home/locators.png`,
            imageAlt: "locators",
            imageAlign: 'left',
        

## Project file reproducing this issue (highly encouraged)

Date/Time: 10/01/2019 00:27 
Branch: master
File: website\pages\en\index.jswebsite\pages\en\index.js

## Environment

OS: Windows 10
Selenium IDE Version:  
Selenium SIDE Runner Version:  
Node version: 10.15.0
Browser: Chrome
Browser Version:  71.0.3578.98 (Official Build) (64-bit)

**Comment by corevo** [comment]
Fix in #531
